linux/vhba_hot_add_remove/wait_device_list_changed.yml

```diff
@@ -143,13 +149,42 @@
 
         - include_tasks: ../utils/reboot.yml
       when: vm_boot_disk_ctrl_type == 'nvme'
-  when:
-    - new_disk_ctrl_type == 'nvme'
-    - vm_hardware_version_num | int < 19
-    - guest_os_ansible_distribution != 'VMware Photon OS'
+
+- name: "Handle NVMe disk / LSILogic disk for FreeBSD"
+  when: 
+    - guest_os_ansible_distribution == "FreeBSD"
+    - new_disk_ctrl_type == 'nvme' or new_disk_ctrl_type == 'lsilogic'
+  block:
+    - name: "Reboot VM to detect NVMe device changes for FreeBSD 13.x or older releases"
+      when: new_disk_ctrl_type == 'nvme'
+      block:
+        - name: "Known issue - workaround of detecting NVMe device changes for FreeBSD"
+          ansible.builtin.debug:
+            msg:
+              - "Guest OS can't detect hot add, remove disk attached to NVMe controller / LSILogic. Ignore this known issue."
```

Comment:
This KB only describes workaround for NVMe controller, no LSILogic.

Reply:
Thanks. Done